python_scripts/01_tabular_data_exploration.py

```diff
@@ -121,7 +130,7 @@
 # without looking much further. We'll do that later in this notebook.
 # * hours per week peaks at 40, this was very likely the standard number of
 # working hours at the time of the data collection
-# * most values of capital-gain and capital-loss are close to zero
+# * some values of capital-gain and capital-loss are far appart from the rest
```

Comment:
This feels a bit too vague. I guess you have outliers in mind whereas I (not sure I wrote this but quite likely) had more in mind this feature may not be that useful since most of the values are zero.

Reply:
Okay, given the histogram, I was not sure that the first bin was all 0. But you're right